lib/internal/Magento/Framework/GraphQl/Exception/GraphQlInputException.php

```diff
@@ -51,4 +60,22 @@ public function getCategory() : string
     {
         return self::EXCEPTION_CATEGORY;
     }
+
+    /**
+     * @param LocalizedException $exception
+     * @return $this
+     */
+    public function addError(LocalizedException $exception): self
```

Comment:
I prefer an immutable type of objects. We can initialization to `construct`. But it's a recommendation but not a requirement.